### PR TITLE
[MMCA-4896] Send email notification for requested CSV cash account transactions

### PR DESCRIPTION
**Pull request by wg-hmrc**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ *@
+
+@(params: Map[String, Any])
+@title = @{"Requested cash account transactions are now available"}
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, title) {
+
+    @uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.salutationLine1(params)
+
+    <div style="font-size: 19px;line-height: 1.315789474;margin: 0 0 30px 0;">
```
Comment (AdamCoops) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.html:24]: Does it matter that we're not using components and its just raw HTML?

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+To access your transactions:
+
+Go to GOV.UK.
+
+Search for 'Use a cash account for Customs Declaration Service declarations'.
+
+For security reasons, we have not included a link in this email.
+
+
```
Comment (AdamCoops) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.txt:15]: Is this double empty line here correct?

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.html (hunk 1 above)
Comment (wg-hmrc) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.html:24]: not for email templates

Comment (AdamCoops) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/customsfinancials/requestedCashAccountTransactions.scala.html:24]: safe

**Review by AdamCoops — APPROVED** [review]

**Review by muralishub — APPROVED** [review]

**Pull request merged**